### PR TITLE
react: fix profile cache manager

### PR DESCRIPTION
**Pull request by krzysu**

### CODE REVIEW

File: packages/react/src/environments.ts [hunk 1]
```diff
@@ -108,35 +108,3 @@ export const development: EnvironmentConfig = {
...
- *
- * This is the environment to be used when you develop and you need to experiment with custom collect/follow modules.
- * Although the Lens contract are also deployed on Mumbai this is a separate deployment so expect different test data, profiles, users, etc.
- *
- * - Endpoint: https://api-sandbox-mumbai.lens.dev
- * - Chain IDs: 80001 (Mumbai), 5 (Goerli)
- * - Profile handle namespace: `test/`
- * - Environment specific timings
- */
-export const sandbox: EnvironmentConfig = {
```
Comment (krzysu) [inline packages/react/src/environments.ts:123]: there won't be sandbox in v2

File: packages/react/src/profile/infrastructure/ProfileCacheManager.ts [hunk 2]
```diff
@@ -52,30 +46,6 @@ export class ProfileCacheManager implements IProfileCacheManager {
     });
   }
 
-  // TODO no longer need to accept a profile id, only authenticated profile can be used
-  updateProfile(id: string, updateFn: (current: Profile) => Profile): void {
```
Comment (krzysu) [inline packages/react/src/profile/infrastructure/ProfileCacheManager.ts:56]: I removed this to promote optimistic implementations using type policies, it was not in use anyway

File: packages/react/src/environments.ts (hunk 1 above)
Comment (cesarenaldi) [reply packages/react/src/environments.ts:123]: no?

File: packages/react/src/profile/infrastructure/ProfileCacheManager.ts (hunk 2 above)
Comment (cesarenaldi) [reply packages/react/src/profile/infrastructure/ProfileCacheManager.ts:56]: 😢

**Review by cesarenaldi — APPROVED** [review]

**Pull request merged**